.gitignore

```diff
@@ -3,3 +3,4 @@
 priv
 
 /client/oracle_client/burrito_out/
+.tool-versions
```

Comment:
It might be good if we set a specific elixir and erlang version

Reply:
I have the same versions, Elixir 14 and Erlang 25, but I use asdf so I have problems by running the application. I modified the tools but I set it into the gitignore for not giving you problems.

Reply:
ok, I'm ok including the `.tool-versions` though if you want. I use asdf as well